dashboard/app/assets/stylesheets/scaffolds.scss

```diff
@@ -33,8 +33,11 @@ div {
   display: inline-block;
 }
 
-.field_with_errors input {
-  background-color: $realyellow;
+.field_with_errors {
```

Comment:
does this change affect other inputs? Also can you include a screenshot of it?

Reply:
Screenshots updated. Hard to know all possible inputs this could be affecting, but it should be a good change because it makes it consistent with the /users/signup styling. Also it's a good a11y pattern to add a border around inputs which are in error.